src/components/Datepicker/Month.tsx

```diff
@@ -61,7 +62,7 @@ const Month: FC<MonthProps> = ({ year, month, firstDayOfWeek, locale }: MonthPro
                         return <Day date={day.date} key={day.date.toString()} dayLabel={day.dayLabel} />;
                     }
 
-                    return <div key={day} />;
```

Comment:
do we understand why the day doesn't work? shouldn't we just create a timestamp for those dates and use them as a key? They are stable

Reply:
The issue is that when a month does not start on monday the calendar renders empty cells to represent the days of the last month included in the current month's calendar (check out the second image in the description). In that case `day` is not a date and we cannot use it as a unique key (I don't remember if `day` equals 0 or `undefined`, but either way we cannot use it).